community/sway/etc/sway/definitions

```diff
@@ -24,9 +24,11 @@ set $background /usr/share/backgrounds/manjaro-sway/gruvbox-dark-manjarosway-384
 set $theme /usr/share/sway/themes/matcha-green
 
 # Your preferred terminal emulator
-set $term /usr/share/sway/scripts/terminal.sh
-set $term_float $term float
-set $term_float_portrait $term float_portrait
+set $term foot
+set $term_client footclient
+set $term_server foot --server
```

Comment:
```suggestion
```
i'd prefer this not to be a variable that is only being used once, because I don't see a scenario where sb would want to override this. and instead will only complicate the readability of the autostart config.

Reply:
Agreed